api/src/main/java/org/commonjava/indy/content/StoragePathCalculator.java

```diff
@@ -25,4 +25,6 @@
 public interface StoragePathCalculator
 {
     String calculateStoragePath( StoreKey storeKey, String path );
+
+    String calculateStoragePath( StoreKey storeKey, String path, boolean isRawView);
```

Comment:
According to the NPMStoragePathCalculator, I see if isRaw is true it just returns the raw path. Basically this new api method don't make more sense than saying "ignore me". Can we hide the details in the old method impl class without adding this new api? I don't dig deep, but it sounds difficult to write a description for the new api. This makes me doubt whether we need it.

Reply:
Um... sounds ok to me. We can wrap that ThreadContext directly in to NPMStoragePathCalculater. Let me try.

Reply:
Done. Rolled back to previous version.